examples/src/main/scala/udf/Gen.scala

```diff
@@ -0,0 +1,38 @@
+package udf
```

Comment:
Maybe `examples` is not the correct submodule for this.
It could in udf/resources

Reply:
moved it